platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+or removal of content, etc. If any material changes are needed, the PR must be closed, and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR.
+
+---
+
+## Voting on A Proposal
+
+Votes follow the [Apache Voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions)
+scheme. Votes are cast in comments on the PR on a range between -1 to -0, +0 to +1 with the following semantics:
+
+* -1: is a veto.
+* -0 >= x > -1: is a vote generally against the proposal.
+* +0 <= x < +1: is a vote generally in favor of the proposal.
+* +1: is a vote in favor of the proposal.
+
+The magnitude of the numeric value of the vote indicates the strength of the sentiment behind the vote.
+
++1 votes can also be expressed by approving the PR. -1 votes can be expressed by "Requesting Changes" on the PR and
+must be accompanied by a comment explaining the reason for the veto.
+
+---
+
+## Acceptance of A Proposal
+
+A proposal becomes `Accepted` when the following criteria have been fulfilled:
+
+1. The proposal has been in the Voting state for at least 3 business days. Business days are defined as SwirldsLabs
+   working days and excludes weekends and company holidays.
+2. The proposal must have at least three +1 votes from platform code owners.
+3. The proposal must not have any -1 votes from platform code owners or architects.
+
+Once accepted, the proposal must be moved to the Accepted status in
+the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board and implementation of the
+proposal may begin.
+
+---
+
+## Superseding A Proposal
+
+A proposal is superseded by a new proposal when material changes are needed once voting has started. The old proposal
+must link to the new proposal and indicate that it is being superseded. After the proposal has been updated, the PR's
+status in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project should be
+changed to `Superseded`.
+
+---
+
+## Withdrawing A Proposal
+
+A proposal may be withdrawn after voting occurs. The proposal must be updated with a reason for the withdrawal. After
+the proposal has been updated, the PR's status in
+the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project should be changed
+to `Withdrawn`.
+
+---
+
+## Delivery of A Proposal
+
+Once an accepted proposal has been completely implemented, tested, the code merged into `develop`, and the feature is
+enabled for production, the proposal's content should be merged with the documentation of the platform
+in `platform-sdk/docs` and removed from `platform-sdk/docs/proposals`. Once the platform documentation is updated, the
+status of the proposal PR in the proposal project should be changed to `Delivered`.
+
+## Design Proposal State Machine
+
+![](designProposalStateMachine.svg)
```

Comment:
Can a proposal that is in accepted state withdrawn or superseded?

Reply:
Good questions. I think....

An accepted proposal becomes documentation of how something will be implemented. That documentation can be amended, but it must go through the voting process like the original proposal did.

An accepted proposal can be withdrawn, but the PR to remove it must go through the voting process like the original and explain why.

Reply:
> but the PR to remove it 

Oh, I have not assumed that we delete proposals.

Reply:
If it's being withdrawn, why not? I hand't considered a proposal being withdrawn after being accepted prior to your question. But i think it would be wrong to leave a proposal in the repo that we know we are not going to do.